backend/tournesol/views/preview.py

```diff
@@ -0,0 +1,95 @@
+import logging
+from io import BytesIO
+
+import requests
+from django.http import FileResponse, HttpResponse
+from django.utils.decorators import method_decorator
+from drf_spectacular.utils import OpenApiTypes, extend_schema
+from PIL import Image, ImageDraw, ImageFont
+from rest_framework.views import APIView
+
+from tournesol.models.entity import Entity
+from tournesol.utils.cache import cache_page_no_i18n
+
+logger = logging.getLogger(__name__)
+
+
+class DynamicWebsitePreviewDefault(APIView):
+    permission_classes = []
+
+    @method_decorator(cache_page_no_i18n(3600 * 24))  # 24h cache
+    @extend_schema(
+        description="Default website preview",
+        responses={200: OpenApiTypes.BINARY},
+    )
+    def get(self, request):
+        return DynamicWebsitePreviewDefault.default_preview()
+
+    @staticmethod
+    def default_preview():
+        default_preview = open("tournesol/resources/tournesol_screenshot_og.png", "rb")
```

Comment:
as discussed in a separate thread, `BASE_DIR` should be used to build relative path:
```suggestion
        default_preview = open(BASE_DIR / "tournesol/resources/tournesol_screenshot_og.png", "rb")
```

Reply:
Fixed by the review branch https://github.com/tournesol-app/tournesol/pull/1092